transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/internal/wizards/TransformTestWizardPage.java

```diff
@@ -484,12 +474,13 @@ private void doStatusUpdate() {
             }
         }
 
-        if (fPackageStatus.getCode() == StatusInfo.ERROR && (fPackageStatus.getMessage() == null
-                || fPackageStatus.getMessage().trim().isEmpty())) {
-            // override this particular case, since the default package is ok, though not great
-            fPackageStatus = new StatusInfo(NONE, null);
-        } else if (fPackageStatus.getCode() == StatusInfo.WARNING && fPackageStatus.getMessage() != null
-                && fPackageStatus.getMessage().contains("default package is discouraged")) { //$NON-NLS-1$
+        boolean defaultPkg = fPackageStatus.getCode() == StatusInfo.ERROR && (fPackageStatus.getMessage() == null
+            				 || fPackageStatus.getMessage().trim().isEmpty());
+        if (!defaultPkg) {
+        	defaultPkg = fPackageStatus.getCode() == StatusInfo.WARNING && fPackageStatus.getMessage() != null
+                		 && fPackageStatus.getMessage().contains("default package is discouraged"); //$NON-NLS-1$
```

Comment:
Okay, so looking into this more, changing the message regarding the default package is easy, but there's another case of this same issue just a few lines above with the container (i.e., folder) not existing.  As an aside, I looked for another API to accomplish this, and there isn't a direct non-private one, although I could effectively jump through some code hoops to reproduce the same logic.  But back to this other issue, the message takes an argument, so I have no easy way to compare the formatted output without recreating that argument.  So, the only option I see is to basically override the containerUpdated() method, which is not small, and change its behavior to skip the cases we want ignored.  Does everyone think this is all worth it?  Keep in mind all I was trying to address were a couple of "simple" Sonar issues...